build.gradle

```diff
@@ -167,22 +167,27 @@ eclipseClasspath {
     }
 }
 
+def isTravis = System.getenv("TRAVIS") != null
+def isAppVeyor = System.getenv("APPVEYOR") != null
+def isCi = isTravis || isAppVeyor
+
 task createConfigs {
     description "Sets up the project by obtaining necessary files and configurations points."
     group "Setup"
+    def testTemplateFilename = isCi ? "test.ci.properties" : "test.template.properties"
```

Comment:
> Actually, what is the purpose of separate test.properties template?

Did you mean `test.template.properties`? That's used to disseminate changes to the `test.properties` (e.g. adding a new field) to existing developers. The latter is not version controlled.